app/src/main/res/values/strings.xml

```diff
@@ -1043,6 +1043,8 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
 
     <string name="quest_gritBinSeasonal_title">Is this also here outside of winter?</string>
 
+    <string name="quest_excrement_bag_dispenser_bin_title">Does this excrement bag dispenser have a bin?</string>
```

Comment:
Maybe being a bit more colloquial is okay/preferable?

```suggestion
    <string name="quest_excrement_bag_dispenser_bin_title">Does this poop bag dispenser have a bin?</string>
```

Reply:
Sure, this is fine for me too